config/software/libpcap.rb

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright:: Copyright (c) 2012-2014 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name "libpcap"
+default_version "1.7.4"
+
+version "1.7.4" do
+  source md5: "b2e13142bbaba857ab1c6894aedaf547"
+end
+
+source url: "http://www.tcpdump.org/release/libpcap-#{version}.tar.gz"
+
+relative_path "libpcap-#{version}"
+
+dependency "flex"
+dependency "bison"
+
+# we omit the omnibus path here because it breaks mac_os_x builds by picking up the embedded libtool
+# instead of the system libtool which the zlib configure script cannot handle.
```

Comment:
Is this behavior of `zlib` OSX-specific? If so I'd be for omitting the embedded path only on OSX.